src/app/f2f/fields.js

```diff
@@ -117,13 +96,11 @@ module.exports = {
     items: [
       {
         value: APP.HAS_EXPIRY_DATE.YES,
-        text: APP.HAS_EXPIRY_DATE.YES,
```

Comment:
I guess there is a plan for the translation of this field and that's why we removed the text?

Reply:
Yes the text in the yaml file is accessed via the value - leaving it in would mean translations don't work, and it's superfluous anyway with or without the need to translate

Reply:
Nice so when we introduce the translations for radios we are going to add a key to this object?

Reply:
Yes based on discussions we had last week with using the key:value pairs, but I've left that for the Welsh translation ticket